src/backend/utilities/configuration.ts

```diff
@@ -78,7 +78,14 @@ const twitch = {
 if (!twitch.clientId || !twitch.secret) {
   throw new ConfigurationError('No Twitch client credentials provided');
 }
+const instagram = {
+  clientId: env.INSTAGRAM_CLIENT_ID as string,
```

Comment:
We overlooked it in some places, but now we're trying to be consistent with how we name our environment variables. According to the pattern it should be `CLIENT_ID_INSTAGRAM`